proto/README.md

```diff
@@ -0,0 +1,16 @@
+# Protocol Buffers
+
+This sections defines the types and messages shared across implementations. The definition of the data structures are located in the [core/data_structures](../spec/core/data_structures.md) for the core data types and ABCI definitions are located in the [ABCI](../spec/abci/README.md) section.
+
+## Process of Updates
+
+The `.proto` files within this section are core to the protocol and updates must be treated as such.
+
+### Steps
+
```

Comment:
This section needs to be worked on, but this is how I imagine the process going.

Reply:
How does versioning play into these steps?